content/en/monitors/incident_management/datadog_clipboard.md

```diff
@@ -16,40 +12,38 @@ The Datadog Clipboard is a cross-platform tool for collecting and sharing signal
 
 ## Cross-Page Exploration
 
-The Clipboard works on all pages in Datadog and keeps a record of all graphs copied by an individual user. The Clipboard does not automatically copy query text, event JSON, or other text-based content. 
+The Clipboard works on all pages in Datadog and keeps a record of all graphs copied by an individual user. The Clipboard does not automatically copy query text, event JSON, or other text-based content.
 
 ## Opening the Clipboard
 
 To open the Clipboard, copy any graph and click “Open Clipboard” in the toast.
 
 {{< img src="monitors/incidents/open-clipboard.png" alt="Open a graph in the Clipboard"  style="width:80%;">}}
 
-Or, click “`Cmd/Ctrl + Shift + X` to open” on the minimized Clipboard. 
+Or, click “`Cmd/Ctrl + Shift + K` to open” on the minimized Clipboard.
 
-The Clipboard can also be opened and closed using `Cmd/Ctrl + Shift + X`. To minimize the Clipboard, click the Minimize icon. The minimized Clipboard persists on all pages of Datadog.
+The Clipboard can also be opened and closed using `Cmd/Ctrl + Shift + K`. To minimize the Clipboard, click the Minimize icon. The minimized Clipboard persists on all pages of Datadog.
 
 ## Adding Clips
 
-To add a graph, copy it with `Cmd/Ctrl + C` or click “Copy” in the export menu. Once the Clipboard is open, copied graphs get added automatically. 
+To add a graph, copy it with `Cmd/Ctrl + C` or click “Copy” in the export menu. Once the Clipboard is open, copied graphs get added automatically.
 
-To add a URL, open the Clipboard and click “Add Current URL.”
+To add a URL, open the Clipboard and click “Add current page.”
 
-{{< img src="monitors/incidents/add-dashboard.png" alt="Add a dashboard to the Clipboard"  style="width:80%;">}}
+{{< img src="monitors/incidents/add-page.png" alt="Add a dashboard to the Clipboard"  style="width:80%;">}}
 
 ## Managing Clips
 
-Each item in the Clipboard can be opened, cloned, or deleted; these options are available when you hover over any signal. Opening an item navigates to the link of the original signal. Open the source of any graph (like the dashboard it was clipped from) by clicking the title at the bottom of the item.
+Each item in the Clipboard can be opened, cloned, or deleted; these options are available when you hover over any signal. Opening an item navigates to the link of the original signal. Open the source of any graph (like the dashboard it was clipped from) by clicking the title of the item.
 
 {{< img src="monitors/incidents/managing-clips.png" alt="Manage your clips"  style="width:80%;">}}
 
 The Clipboard holds a maximum of 20 graphs. Remove graphs by deleting them individually, or by clicking “Clear Clipboard” on the bottom left. If more than 20 graphs are added, the oldest graphs, stored furthest to the left, are removed automatically.
```

Comment:
This appears to now be "Remove All" and is in the top right of the clipboard UI.

Reply:
good catch, updated!